tests/samples_tests/smoke_tests/README.md

```diff
@@ -19,7 +19,7 @@ cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -P cmake_install.cm
 a. Required:
     - IE_APP_PATH : coomon path to C++ and C samples, e.g. '<INSTALL_DIR>/samples_bin'
     - IE_APP_PYTHON_PATH : path to python IE samples, e.g. '<INSTALL_DIR>/samples/python/'
-    - IE_APP_PYTHON_TOOL_PATH : path to python IE tools for benchmark_app, e.g. '<INSTALL_DIR>/tools/' 
```

Comment:
why this way does work anymore?

Reply:
This was the only mention of `IE_APP_PYTHON_TOOL_PATH` in the whole repo